src/ReadLine/ReadLine.cs

```diff
@@ -14,13 +14,25 @@ static ReadLine()
             _history = new List<string>();
         }
 
+        /// <summary>
+        /// Will be placed at the end of the prompt. 
+        /// Would you like to continue [Y]:
+        ///                               ^
+        /// </summary>
+        public static string PromptCloser = ":";
+
         public static void AddHistory(params string[] text) => _history.AddRange(text);
         public static List<string> GetHistory() => _history;
         public static void ClearHistory() => _history = new List<string>();
 
-        public static string Read(string prompt = "")
+        public static string Read(string prompt = "", string defaultInput = "")
         {
             Console.Write(prompt);
+            if (!String.IsNullOrWhiteSpace(defaultInput))
+            {
+                Console.Write($"[{defaultInput}]");
+            }
+            Console.Write(PromptCloser);
```

Comment:
No need to print out the default input or prompt closer let the developer specify the output string format using the `prompt` parameter